resources/bundles/org.eclipse.core.filesystem/src/org/eclipse/core/internal/filesystem/local/Convert.java

```diff
@@ -25,8 +25,8 @@ public class Convert {
 	/** Indicates if we are running on windows */
 	private static final boolean IS_WINDOWS = Platform.OS.isWindows();
 
-	private static final String WIN32_FILE_PREFIX = "\\\\?\\"; //$NON-NLS-1$
-	private static final String WIN32_UNC_FILE_PREFIX = "\\\\?\\UNC"; //$NON-NLS-1$
+	public static final String WIN32_LONG_PATH_PREFIX = "\\\\?\\"; //$NON-NLS-1$
```

Comment:
That this can be used for long paths is only an example
`For file I/O, the "\\?\" prefix to a path string tells the Windows APIs to disable all string parsing and to send the string that follows it straight to the file system. For example, if the file system supports large paths and file names, you can exceed the MAX_PATH limits that are otherwise enforced by the Windows APIs`

Reply:
That's right, but I didn't found a technical term for it. Do you know one?
Unless we find a 'definition' from Microsoft I would continue with `WIN32_RAW_PATH_PREFIX`/`WIN32_UNC_RAW_PATH_PREFIX`. The current name basically says nothing.